src/api/contexts/buy.context.tsx

```diff
@@ -10,6 +10,7 @@ import { Buy, BuyPaymentInfo } from '../definitions/buy';
 interface BuyInterface {
   currencies?: Fiat[];
   bankAccounts?: BankAccount[];
+  isLoadingAccount: boolean;
```

Comment:
Is it on purpose `isLoadingAccount` but `isUserLoading`?

Reply:
no not really. will rename account one to `isAccountLoading`